torchao/dtypes/uintx/gemlite_layout.py

```diff
@@ -46,15 +46,14 @@ def get_gemlite_quant_kwargs(bit_width, group_size):
     return kwargs
 
 
-def apply_gemlite_quant(
+def get_gemlite_aqt_kwargs(
     weight,
     group_size=64,
     bit_width=4,
     packing_bitwidth=8,
     contiguous=None,
```

Comment:
this one is using `packing_bitwidth=8` by default, with the other functions are using 32. Ideally this one should be using 32 too

Reply:
will change, looks like it should be OK